themes/nekolight/nekolight.theme.sh

```diff
@@ -0,0 +1,27 @@
+#! bash oh-my-bash.module
+
+_omb_theme_nekolight_version='1.0.0'
+
+_omb_theme_nekolight_symbol=""
+
+function _omb_theme_nekolight_git_info() {
+  if _omb_prompt_git rev-parse --is-inside-work-tree &>/dev/null; then
+    local _omb_prompt_nekolight_branch=$(_omb_prompt_git branch --show-current 2>/dev/null)
```

Comment:
This is a local variable. Local variables do not need to be prefixed.

```suggestion
    local branch=$(_omb_prompt_git branch --show-current 2>/dev/null)
```

The later `${_omb_theme_nekolight_symbol}` should also be updated accordingly.

Reply:
Thanks.